pkgs/os-specific/linux/kvdo/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, lib, fetchFromGitHub, vdo, kernel }:
+
+stdenv.mkDerivation rec {
+  inherit (vdo) version;
+  pname = "kvdo";
+
+  src = fetchFromGitHub {
+    owner = "dm-vdo";
+    repo = "kvdo";
+    rev = version;
+    sha256 = "0lnax2i28zy9lx9067xl45z62qpp5ppwa4jf9gbb12pssnz0irrp";
+  };
+
+  dontConfigure = true;
+
+  KSRC = "${kernel.dev}/lib/modules/${kernel.modDirVersion}/build";
+  INSTALL_MOD_PATH = placeholder "out";
+
+  buildPhase = ''
```

Comment:
I think most kernel module packages just add to makeFlags or maybe buildFlags. Would this not work here?

Reply:
I copied that from `dpdk-kmods`. I figured out a different way now, but specifying `makeFlags` in nix leads to escaping which breaks it, so I'm setting it in `preBuild` (copied from `amdgpu-pro`).

Reply:
Looks tame enough now, yes.